lab-kaylyn/model/portfolio.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const uuid = require('node-uuid');
+const createError = require('http-errors');
+const debug = require('debug')('portfolio:portfolio');
+const storage = require('../lib/storage.js');
+
+const Portfolio = module.exports = function(about, projects, work) {
+  debug('instantiate portfolio');
+  if(!about) throw createError(400, 'expected about');
+  if(!projects) throw createError(400, 'expected projects');
+  if(!work) throw createError(400, 'expected work');
+  this.id = uuid.v1();
+  this.about = about;
+  this.projects = projects;
+  this.work = work;
+  debug('about', this.about);
+  debug('projects', this.projects);
+};
+
+Portfolio.createPortfolio = function(_portfolio) {
```

Comment:
@kaylynyuh great use of the underscore naming convention to differentiate `portfolio` from `_portfolio`